docs/reference/edgectl-download.md

```diff
@@ -0,0 +1,42 @@
+# Downloads
+
+You can download `edgectl`, the executable for Edge Control on a macOS, Linux, or
+Windows operating machine.
+
+On macOS and Linux, you have the option to install `edgectl` via a curl command or by downloading and executing a binary file.
+
+## macOS
+
+1. To download by curl command, run the following in your command line:
+
+```shell
+sudo curl -fL https://metriton.datawire.io/downloads/darwin/edgectl -o /usr/local/bin/edgectl && sudo chmod a+x /usr/local/bin/edgectl
+```
+
+2. Or, download the binary file [here](https://metriton.datawire.io/downloads/darwin/edgectl)
+3. Make it executable with the command `chmod a+x ~/Downloads/edgectl`
```

Comment:
These items (3, 4, 5) should be sub-numbers under 2 otherwise this list renders oddly. It's 1 or 2, and 2 has 2.1, 2.2, 2.3. Listing that as 1, 2, 3, 4, 5 reads as 1 or 2 or 3 or 4 or 5.